benchmark/stargz_utils.go

```diff
@@ -16,13 +16,104 @@
 
 package benchmark
 
+import (
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"syscall"
+	"time"
+)
+
+var (
+	stargzAddress = "/tmp/containerd-stargz-grpc/containerd-stargz-grpc.sock"
```

Comment:
noticed this is here as opposed to in `benchmark/benchmark_tests.go` any reason why you chose a different pattern?

Reply:
It is stargz specific, so I put it in ``stargz_utils.go``.